src/purchases.ts

```diff
@@ -478,23 +532,64 @@ export default class Purchases {
    * Make a purchase
    *
    * @param {PurchasesPackage} aPackage The Package you wish to purchase. You can get the Packages by calling getOfferings
-   * @param {UpgradeInfo} upgradeInfo Android only. Optional UpgradeInfo you wish to upgrade from containing the oldSKU
-   * and the optional prorationMode.
+   * @param {UpgradeInfo} upgradeInfo DEPRECATED. Use googleProductChangeInfo.
+   * @param {GoogleProductChangeInfo} googleProductChangeInfo Android only. Optional GoogleProductChangeInfo you 
+   * wish to upgrade from containing the oldProductIdentifier and the optional prorationMode.
+   * @param {boolean} googleIsPersonalizedPrice Android and Google only. Optional boolean indicates personalized pricing on products available for purchase in the EU.
+   * For compliance with EU regulations. User will see "This price has been customize for you" in the purchase dialog when true.
+   * See https://developer.android.com/google/play/billing/integrate#personalized-price for more info.
    * @returns {Promise<{ productIdentifier: string, customerInfo: CustomerInfo }>} A promise of an object containing
    * a customer info object and a product identifier. Rejections return an error code, a boolean indicating if the
    * user cancelled the purchase, and an object with more information. The promise will be also be rejected if configure
    * has not been called yet.
    */
   public static async purchasePackage(
     aPackage: PurchasesPackage,
-    upgradeInfo?: UpgradeInfo | null
+    upgradeInfo?: UpgradeInfo | null,
+    googleProductChangeInfo?: GoogleProductChangeInfo | null,
```

Comment:
Ugh too bad we can't make this `UpgradeInfo | GoogleProductChangeInfo | null` because of the name change 😢

Reply:
I knnooow... I mean, we _could_ would also feel gross. Really don't like that we can't have duplicate methods. But... @tonidero made a Linear task for moving these to builder pattern like we did in Android which would have made this a ton cleaner 🤷‍♂️  So at least we have that as an option going forward!

Reply:
Yeah, I think moving this to something like what we have in `configure`, which accepts an object is the right way to go, since you can deprecate fields in the object.